transaction-relayer/src/main.rs

```diff
@@ -192,6 +193,10 @@ struct Args {
     /// If any transaction mentions these addresses, the transaction will be dropped.
     #[arg(long, env, value_delimiter = ' ', value_parser = Pubkey::from_str)]
     ofac_addresses: Option<Vec<Pubkey>>,
+
+    /// Webserver bind address that exposes diagnostic information
+    #[arg(long, env, default_value_t = SocketAddr::from_str("127.0.0.1:11227").unwrap())]
+    webserver_bind_addr: SocketAddr,
```

Comment:
rename to `health_server_bind_addr`?

Reply:
prefer to keep web just in case we add more functionality to it. maybe something more generic?

Reply:
`admin_server_bind_addr`?